cli/command/stack/swarm/deploy_composefile.go

```diff
@@ -256,10 +264,29 @@ func deployServices(ctx context.Context, dockerCli command.Cli, services map[str
 				createOpts.QueryRegistry = true
 			}
 
-			if _, err := apiClient.ServiceCreate(ctx, serviceSpec, createOpts); err != nil {
-				return errors.Wrapf(err, "failed to create service %s", name)
+			response, err := apiClient.ServiceCreate(ctx, serviceSpec, createOpts)
+			if err != nil {
+				return errors.Wrapf(err, "Failed to create service %s", name)
```

Comment:
These errors should probably remain lowercase (go convention it to have lowercase errors). (If needed, we can still add some prefix when printing)

Reply:
Reverted it!